src/VisualStudio/LiveShare/Impl/Shims/TypeScriptHandlerShims.cs

```diff
@@ -76,7 +75,8 @@ internal class TypeScriptCompletionResolverHandlerShim : CompletionResolveHandle
 
         [ImportingConstructor]
         [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-        public TypeScriptCompletionResolverHandlerShim(ILspWorkspaceRegistrationService workspaceRegistrationService) : base(completionListCache: null)
+        public TypeScriptCompletionResolverHandlerShim(ILspWorkspaceRegistrationService workspaceRegistrationService)
```

Comment:
Also, if we pass in a completionListCache here in the resolve handler, can we do the same in `TypeScriptCompletionHandlerShim`?